### PR TITLE
{chem}[intel/2015a] ABINIT 7.10.4 using included deps (netCDF, libxc) (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

File: easybuild/easyconfigs/a/ABINIT/ABINIT-7.10.4-intel-2015a-incl-deps.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ electronic structure of systems made of electrons and nuclei (molecules and periodic solids) within Density Functional
+ Theory (DFT), using pseudopotentials and a planewave or wavelet basis."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+source_urls = ['http://ftp.abinit.org/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['bfc76a5f93c3f148c049f57541864e76']
+
+configopts = "--with-mpi-prefix=$EBROOTIMPI/intel64 --with-trio-flavor='etsf_io+netcdf' "
```
Comment (pescobar) [inline easybuild/easyconfigs/a/ABINIT/ABINIT-7.10.4-intel-2015a-incl-deps.eb]: isn't there any other variable which is not dependent on the intel toolchains?

Comment (boegel) [inline easybuild/easyconfigs/a/ABINIT/ABINIT-7.10.4-intel-2015a-incl-deps.eb]: we only have `$MPI_LIB_DIR` and `$MPI_INC_DIR`, so no, for now

what we would need here is `$MPI_BIN_DIR`, and then we can use `$MPI_BIN_DIR/../` reliably here

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/1bbe130f59e7f111886b for a full test report.

**Comment by boegel** [comment]
Thanks for the review @pescobar!

**Pull request merged**